File: java/src/main/java/com/opl/aircraftperformancecalculator/models/AirportWeather.java
package com.opl.aircraftperformancecalculator.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class AirportWeather {

    @Id
    private String airportID;

    private Double temp;
    private Double pressureAltitude;
    private Double headWind;
    private Double precipitation;
}

File: FrontEnd/src/app/models/airport.weather.ts
export class AirportWeather {
    public airportID: string;
    public temp: number;
    public pressureAltitude: number;
    public headWind: number;
    public precipitation: number;

    constructor(airportID: string, temp: number, pressureAltitude: number, headWind: number, precipitation: number) {
        this.airportID = airportID;
        this.temp = temp;
        this.pressureAltitude = pressureAltitude;
        this.headWind = headWind;
        this.precipitation = precipitation;
    }
}